### ISSUE TITLE
Soundtrack deep integration

### ISSUE THREAD

**Issue by delahee** [itchio/itch.io#811]
Hello guys, we plan to release our soundtrack on Itch, I haven't found documentations about best practices about that, if you have some please lemme know, otherwise some suggestions

- detect m3u playlist 
- make a simple music player 
- detect channels like 'mp3', 'ost', 'ogg'

And that's it for me :)

Thanks a lot lemme know if the topic interests y'all! Bye And thank you for the cool work!

**Comment by fasterthanlime** [comment]
Hey David!

That's a feature I'd really like to add, but don't have the time to do address right now :)

In the meantime, it's safe to assume that whenever we do come up with a soundtrack player, it'll recognizes m3u playlists and support both mp3 & ogg (at least!). Just go with your best judgement, we'll do as we usually do and analyze the already-uploaded content so that our solution works with as much existing content as possible.

**Comment by delahee** [comment]
Yep no problem and no emergency feel free to close whenever you want. I
removed my current m3u as pathes were absolute I'll dig further after
release to hand edit one. Bye

Le mer. 4 avr. 2018 à 06:02, Amos Wenger <notifications@github.com> a
écrit :

**Comment by fasterthanlime** [comment]
> I'll dig further after release to hand edit one.

That's probably not really necessary - just a folder full of .mp3/.ogg is a pretty good indicator we're dealing with an album :)
